.travis.yml

```diff
@@ -0,0 +1,24 @@
+# dist: trusty  # Ubuntu 14.04
+language: python
+python: 3.6
+branches:
+  only:
+  - gh-pages
+  - /.*/
+before_install:
+  - sudo apt-key adv --keyserver keyserver.ubuntu.com --recv-keys E084DAB9
+  - echo "deb https://cran.rstudio.com/bin/linux/ubuntu trusty/" | sudo tee -a /etc/apt/sources.list
+  - sudo apt-get update -y
+  - sudo apt-get install -y r-base
```

Comment:
I am wondering if the installation of R can be either become conditional (if _episodes_rmd is empty, skip installing R) or the entire setup can be put into a container. I guess, I'll ask in the styles repo.

Reply:
Excellent point. Since the HPC Carpentry episodes do not currently depend on R, I have removed those dependencies from `.travis.yml`, and the CI build now runs in about a minute. Thanks!